app/src/main/java/app/inf1d_team3/open_day/AskQuestionActivity.java

```diff
@@ -0,0 +1,52 @@
+package app.inf1d_team3.open_day;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+
+public class AskQuestionActivity extends AppCompatActivity {
+
+    // messege
+    private EditText mEditTextSubject;
+    private EditText mEditTextMessage;
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.ask_question_activity);
+
+        // messege
+        mEditTextSubject = findViewById(R.id.edit_text_subject);
+        mEditTextMessage = findViewById(R.id.edit_text_message);
+
+        Button buttonSend = findViewById(R.id.button_send);
+        buttonSend.setOnClickListener(new View.OnClickListener() {
```

Comment:
Change onclick to be connect via xml instead of via the code (better compatability)